iceoryx_utils/platform/mac/include/iceoryx_utils/platform/acl.hpp

```diff
@@ -33,17 +33,17 @@ using acl_perm_t = int;
 using acl_entry_t = int;
 using acl_tag_t = int;
 
-inline int acl_valid(acl_t acl)
+inline int acl_valid(acl_t)
```

Comment:
it's preferred to use [[gnu::unused]] instead of deleting the parameter name, especially if there is a function like foo(int, int, int)

Reply:
You are right in general but in the case of the Mac OS posix platform implementation where we just provide empty functions so that the code compiles I think it adds a lot of code bloat with nothing to gain. If we would have an underlying implementation than I would agree with you but since they are only empty stub functions it is not of any advantage when we know what parameter represents what since we are always `return 0;`